### ISSUE TITLE
Feature: Detect if the user have read receipts on

### ISSUE THREAD

**Issue by pedrogrisolia** [open-wa/wa-automate-nodejs#1056]
I don't know what to write here lol.
It would be nice to have a method to detect that, or put a property on the Message object that would be true or false, if the user have read receipts on or off.

Thanks!

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.0.15](https://github.com/open-wa/wa-automate-nodejs/compare/3.0.14...3.0.15) (2020-11-11)

- build(deps-dev): bump @types/express from 4.17.8 to 4.17.9 [`#1060`](https://github.com/open-wa/wa-automate-nodejs/pull/1060)
- build(deps): bump @types/sharp from 0.26.0 to 0.26.1 [`#1053`](https://github.com/open-wa/wa-automate-nodejs/pull/1053)
- build(deps-dev): bump @types/node from 14.14.6 to 14.14.7 [`#1054`](https://github.com/open-wa/wa-automate-nodejs/pull/1054)
- build(deps): bump socket.io from 2.3.0 to 3.0.0 [`#1044`](https://github.com/open-wa/wa-automate-nodejs/pull/1044)
- build(deps): bump swagger-ui-dist from 3.36.1 to 3.36.2 [`#1048`](https://github.com/open-wa/wa-automate-nodejs/pull/1048)
- Release 3.0.15 [`40c1c94`](https://github.com/open-wa/wa-automate-nodejs/commit/40c1c949007af0038427e983396538b2132b3837)
- Insiders: `checkReadReceipts` #1056 [`2418884`](https://github.com/open-wa/wa-automate-nodejs/commit/241888413ae8cc3db73d7cc0df2287bc5843e668)
- Feature Unlocked: `setGroupToAdminsOnly` now GA #970 [`4ba49fa`](https://github.com/open-wa/wa-automate-nodejs/commit/4ba49fa34b6bb4c0bf2570425f0d81543d2c3bae)
- patch: possible fix for #1059 [`2345eff`](https://github.com/open-wa/wa-automate-nodejs/commit/2345efff6abcfc00d6e353c3645f816724a0484a)
- patch for #1055 [`44ffc24`](https://github.com/open-wa/wa-automate-nodejs/commit/44ffc2491e5a15210e29fa7f4056b01dc60d8b24)
- patch: #1057 [`fcd0b13`](https://github.com/open-wa/wa-automate-nodejs/commit/fcd0b13603935f3d88bf9fe6a13ae271235cbfef)
- Critical patches update [`e36bbc6`](https://github.com/open-wa/wa-automate-nodejs/commit/e36bbc6a4a35260df10aa1596569d58c2a683076)
- fix typo in CLI docs [`f81820e`](https://github.com/open-wa/wa-automate-nodejs/commit/f81820ef1402519a4bb53fbdd02f84307d2ec8f2)
- Feature Unlocked: `setGroupEditToAdminsOnly` now GA #970 [`e7eea7d`](https://github.com/open-wa/wa-automate-nodejs/commit/e7eea7d51683f31c0c9e8e41f34bb89c3333668d)
- Feature Unlocked: `setProfilePic` now GA [`04c2e36`](https://github.com/open-wa/wa-automate-nodejs/commit/04c2e36b545279a00999e20bb5c43b7323463726)
- fix: host notifications patched [`9120626`](https://github.com/open-wa/wa-automate-nodejs/commit/912062675bdb7a6b6b909ec8fd5b4a3c74b924e6)
- added error when `create` has a string first param #1046 [`7092104`](https://github.com/open-wa/wa-automate-nodejs/commit/70921042c7a47c2f276653ad45a80d2b0b6f5fe8)

**Comment by smashah** [comment]
https://open-wa.github.io/wa-automate-nodejs/classes/client.html#checkreadreceipts
